src/test/java/io/cdap/plugin/gcp/spanner/connector/SpannerConnectorTest.java

```diff
@@ -160,10 +160,11 @@ private void testGenerateSpec(SpannerConnector connector) throws IOException {
     properties.put("table", table);
     properties.put("connection", null);
     properties.put("useConnection", "true");
```

Comment:
Can we rename this as `sinkProperties` like we did in the other file to avoid confusion?

Reply:
The reason I didn't was that properties is like the parent, and source diverges further down but before that it's all shared between the two. maybe I thought about it too much.

Reply:
Right. But if I were to look at the code and I saw a `sourceProperties` and a `sinkProperties` it would be clearer to me than if it were `sourceProperties` and `properties`